cfgmassif.go

```diff
@@ -24,14 +24,29 @@ func cfgMassifReader(cmd *CmdCtx, cCtx *cli.Context) error {
 		}
 	}
 
-	if cmd.reader == nil {
-		if err = cfgReader(cmd, cCtx); err != nil {
+	logdir := cCtx.String("logdir")
```

Comment:
Why not use an argv positional argument and just test if it is a file or a directory ? Its more natural on the command line I think

Reply:
so I think we already use positional args for event.json and what not - then the order would start to matter too and it's all potentially bit messy - guess we should probably agree on a sensible approach though that will work for all the commands